packages/k10/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,12 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -8,3 +8,8 @@
+   name: kastenIO
+ name: k10
+ version: 4.5.6
++kubeVersion: ">= 1.17"
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: k10
++  catalog.cattle.io/display-name: kasten/k10 
+\ No newline at end of file
```

Comment:
```suggestion
\ No newline at end of file
```
Should this be present? How was this generated?

Reply:
removed that. I did not add a new line to the original file which was used to generate this.